TrafficCapture/captureKafkaOffloader/build.gradle

```diff
@@ -0,0 +1,27 @@
+
+plugins {
+    id 'org.opensearch.migrations.java-library-conventions'
+    id("io.freefair.lombok") version "8.0.1"
+}
+
+repositories {
+    mavenCentral()
+}
+
+sourceSets {
+    main {
+        java {
+            srcDirs 'src/main/java'
+        }
+    }
+}
+dependencies {
+    api group: 'io.netty', name: 'netty-buffer', version: '4.1.89.Final'
+    implementation project(':captureOffloader')
```

Comment:
I'd say, please pick one addressing format over the other, but we shouldn't probably pick one for the entire solution & I'm torn between the two.